### DISCUSSION TITLE
One more touchscreen and enclosure option

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#300] [Ideas]
This looks like an interesting option for DIY nav station computer 

https://m.aliexpress.us/item/3256803892065315.html?fbclid=IwAR26M9qO9vPDn87nifru4jt3iI9SaVTZn4R4JXO00YRH8jXlvD8-INnLGmE&gatewayAdapt=gloPc2usaMsite4itemAdapt&mibextid=Zxz2cZ

**Comment by mgrouch** [comment]
Re:Terminal DM in 10” form

https://wiki.seeedstudio.com/reterminal-dm/

**Reply by jennazhuxj** [reply]
FYI: just saw the updated datasheet for this reTerminal DM: https://files.seeedstudio.com/wiki/reTerminalDM/reTerminalDM_datasheet.pdf. saying the reTerminal DMs are under strict vibration and shock testing, suitable for automotive, and marine applications.
